cephfs/admin/volume_test.go

```diff
@@ -384,10 +390,19 @@ func TestListFileSystems(t *testing.T) {
 
 	l, err := fsa.ListFileSystems()
 	assert.NoError(t, err)
-	if assert.Len(t, l, 1) {
-		assert.Equal(t, "cephfs", l[0].Name)
-		assert.Equal(t, "cephfs_metadata", l[0].MetadataPool)
-		assert.Len(t, l[0].DataPools, 1)
-		assert.Contains(t, l[0].DataPools, "cephfs_data")
+	assert.GreaterOrEqual(t, len(l), 1)
+
+	idx := -1
+	for i := range l {
+		if l[i].Name == "cephfs" {
+			idx = i
+			break
+		}
+	}
+	if assert.NotEqual(t, -1, idx) {
```

Comment:
another nit (and the code was like that before): wouldn't that be a case for `require.NotEqual()`? I never noticed, that `assert` returns a bool, and I wonder why.

Reply:
You could use require for this but I didn't feel like mixing both assert and require in this case.
I'm pretty sure using the assert calls this way is documented somewhere at testify.